test/exercism/homework_test.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../test_helper'
+require_relative '../active_record_helper'
+require_relative '../../lib/exercism'
+require_relative '../../lib/exercism/homework'
+
+class HomeworkTest < Minitest::Test
+  include DBCleaner
+
+  def alice
+    @alice ||= User.create(username: 'alice')
+  end
+
+  def teardown
+    super
+    @alice = nil
+    @language = nil
+    @skipped = nil
+    @fetched = nil
+    @recent = nil
+  end
+
+  def create_exercises
+    @language = 'ruby'
+    time = Time.now
+
+    @skipped = UserExercise.create(user: alice,
+                                   language: @language,
+                                   slug: 'leap',
+                                   skipped_at: time)
+
+    @fetched = UserExercise.create(user: alice,
+                                   language: @language,
+                                   slug: 'clock',
+                                   fetched_at: time)
+
+    @recent = UserExercise.create(user: alice,
+                                  language: @language,
+                                  slug: 'gigasecond',
+                                  last_iteration_at: time)
+  end
+
+  def test_status_with_progress_on_track
+    create_exercises
+    status = Homework.new(alice).status(@language)
+
+    assert_equal status, {
+      track_id: @language,
+      recent: {
+        problem: @recent.slug,
+        submitted_at: @recent.last_iteration_at
+      },
+      skipped: [@skipped.slug],
+      fetched: [@fetched.slug]
+    }
+  end
+
+  def test_status_with_no_progress_on_track
```

Comment:
You don't need two tests for this, just create one test and make both assertions there.